utils/cLog.h

```diff
@@ -62,5 +64,7 @@ class cLogger {
 
       long startTime; // start time of logging in miliseconds
 
+      bool m_debugMode;
```

Comment:
Eventually, we could let the debug mode be dependent on the log level, but that's not easy to do half-half. We'll need to decide that and change all the calls at once, I expect.

Reply:
Hmm. Well, in the end `debugMode` basically enables some 'flags' to allow us to see (visually, like drawing on screen) different things. It *also* has impact on logging atm.

So those things need to be untangled:
- log levels (like you said)
- debug/drawing information per feature perhaps? (with some kind of variables)

For some other time for sure :)